theme/scripts/cart.ts

```diff
@@ -216,7 +216,7 @@ export default(function() {
       }
 
```

Comment:
Line 209, above: would it be worth adding an `Action.UPDATE` and making the action argument required?

(it looks like it's required in `handleAddtoCartButton`... since it's optional in `onChange` I'd expect TS to be bugging you about this?)